SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -78,6 +77,8 @@ - (void)sdl_startEventListening {
                                              selector:@selector(sdl_applicationWillEnterForeground:)
                                                  name:UIApplicationWillEnterForegroundNotification
                                                object:nil];
+    
+    [[EAAccessoryManager sharedAccessoryManager] registerForLocalNotifications];
```

Comment:
Just want to double check that we don't need to balance this with an `unregister`. It doesn't appear to be required, but this `register` may be called several times if the app registers and unregisters.